src/csclng.c

```diff
@@ -37,6 +37,8 @@ const char *wrapper_debug_envvar_name = "DEBUG_CSCLNG";
 
 const char *analyzer_name = "clang";
 
+const char *analyzer_bin_envvar = NULL;
```

Comment:
Could you please name the new symbol `analyzer_bin_envvar_name`?  It would be more consistent with the existing symbols `wrapper_addopts_envvar_name` and `wrapper_debug_envvar_name`.